core/system/src/mpu/vmpu_armv7m.c

```diff
@@ -703,7 +703,7 @@ void vmpu_arch_init_hw(void)
     vmpu_acl_static_region(
         2,
         (void *) SRAM_ORIGIN,
-        ((uint32_t) __uvisor_config.bss_end) - SRAM_ORIGIN,
+        ((uint32_t) __uvisor_config.os_priv_end) - SRAM_ORIGIN,
```

Comment:
@Patater - as discussed bss_end needs to stay bss_end (os_priv_end not correct).